src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -223,15 +223,13 @@ uct_ud_verbs_ep_am_zcopy(uct_ep_h tl_ep, uint8_t id, const void *header,
     memcpy(skb->neth + 1, header, header_length);
     skb->len = sizeof(uct_ud_neth_t) + header_length;
 
-    iface->tx.sge[1].lkey   = (ib_memh == UCT_INVALID_MEM_HANDLE) ? 0 : ib_memh->lkey;
-    iface->tx.sge[1].length = length;
-    iface->tx.sge[1].addr   = (uintptr_t)buffer;
-    iface->tx.wr_skb.num_sge = 2;
+    iface->tx.wr_skb.num_sge = uct_rc_verbs_rdma_zcopy_sge_fill_iov(&(iface->tx.sge[1]),
```

Comment:
also, we should rename "uct_rc_verbs_rdma_zcopy_sge_fill_iov" to "uct_ib_verbs_sge_fill_iov"